MediaAdmin/EventSubscriber/MediaDeletedSubscriber.php

```diff
@@ -41,7 +41,10 @@ public function deleteAlternatives()
     {
         /** @var MediaInterface $media */
         foreach ($this->medias as $media) {
-            $this->fileAlternativesManager->deleteThumbnail($media);
+            if (null !== $media->getThumbnail()) {
```

Comment:
I think, It is not a good idea whatsoever the manager who decides to launch the strategies or not. for instance a developer wants to be a strategy that puts a default image in the case where there aren't thumbnail